packages/zone.js/lib/zone.ts

```diff
@@ -678,7 +678,7 @@ type AmbientZoneDelegate = ZoneDelegate;
 // This code should run in a Browser, so we don't want to include the whole node.d.ts
 // typings for this compilation unit.
 // We'll just fake the global "global" var for now.
-declare var global: NodeJS.Global;
+declare var global: any;
```

Comment:
```suggestion
declare var global: NodeJS.Global & typeof globalThis;
```
Make sure that the global type match the one provided by the current `@types/node` module.